index.html

```diff
@@ -7643,6 +7643,51 @@ <h2 id="WaveShaperNode">
         </dl>
         <section>
           <h2>
+            OscillatorOptions
+          </h2>
+          <p>
+            This specifies the options to be used when constructing an
+            <a><code>OscillatorNode</code></a>. All of the members are optional;
+            if not specified, the normal default values are used for
+            constructing the oscillator.
+          </p>
+          <dl title="dictionary OscillatorOptions : AudioNodeChannelOptions"
```

Comment:
Does it make sense for `OscillatorOptions` to derive from `AudioNodeOptions`?  The channel stuff really only applies to inputs and this node has no inputs at all. Setting them will change the values for the node, as they do today, but they have no effect on the node itself.

Leave it for a kind of backward compatibility?